src/django_components/component_media.py

```diff
@@ -43,7 +45,7 @@
 
 
 # These are all the attributes that are handled by ComponentMedia and lazily-resolved
-COMP_MEDIA_LAZY_ATTRS = ("media", "template", "template_file", "js", "js_file", "css", "css_file")
+COMP_MEDIA_LAZY_ATTRS = ("media", "template", "template_file", "js", "js_file", "css", "css_file", "_template")
```

Comment:
Accesing `Component._template` now behaves like the other "media attributes". that is, accessing this attribute triggers lazy loading of the HTML / JS / CSS, if not loaded / resolved yet.